data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+
+
+
+mission "James Medicine [0]"
+	minor
+	name "Medicine from <planet>"
+	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
+	deadline
+	source "Hestia"
+	destination "Nimbus"
+	to offer
+		has "main plot completed"
+		or
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Interceptor]: done"
+	on offer
+		conversation
+			`Finding yourself on <origin>, you smile at the memory of having dropped off James here after he gave you some basic spaceflight instructions. Do you wish to visit him and see how he has been faring?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	Unlike some other planets, where it is practically impossible to find a person without having contact information, <origin> has a planetary database. You eagerly walk over to the nearest information desk and wait for the operator to make a few calls. At last, you are directed to a small, respected establishment, the Pilot Veteran Clinic. You can only wonder what James is doing there.`
+			`	The clinic is housed in a small, modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`
+			`	There is no one in sight at the clinic, probably because it is around lunch time. Luckily, the operator gave you the number of James's room, and you manage to find the correct door after a few minutes.`
+			choice
+				`	(Wait outside for someone to show up.)`
+					goto wait
+				`	(Enter the room.)`
+			apply
+				set "james medicine: infected"
+			`	You enter the room and find that James is inside, lightly asleep. He wakes up just as you close the door.`
+			`	"<first>! Fancy meeting you here! I heard you became a war hero of sorts?", he adds with a wink.`
+			choice
+				`	"Nothing special, I just did what is right."`
+				`	"Quite so! Seems like I saved the galaxy this time."`
+			`	"Yeah, yeah... You know, I am feeling pretty weak now, and while the doctors say that I will live through this illness, there is still a chance that it would be my last."`
+			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but.. there is a wormhole there!"`
+			branch interesting
+				not "First Contact: Hai: offered"
+			choice
+				`	(Tell him about the Hai.)`
+				`	(Do not mention the Hai.)`
+					goto interesting
+			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
```

Comment:
```suggestion
			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
```

Reply:
Uhh, should something be changed here?

Reply:
I'm not sure about putting a period there; going to check other conversations and grammar sites...